internal/backends/python/pypi_map.override.go

```diff
@@ -9,5 +9,6 @@ var moduleToPypiPackageOverride = map[string]string{
 	"requirements": "requirements-parser", // popular rlbot depends on it, but doesn't supply requires_dist
 	"base62":       "pybase62",            // it was overridden by base-62 which wins due to name match but is less popular by far
 	"faiss":        "faiss-cpu",
-	"graphics":     "graphics.py",         // this package is popular, but the module doesn't match the package name https://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/graphics.html#a-graphics-introduction
+	"graphics":     "graphics.py", // this package is popular, but the module doesn't match the package name https://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/graphics.html#a-graphics-introduction
+	"replit.ai":    "replit-ai",   // Replit's AI package
```

Comment:
```suggestion
	"replit.ai":    "replit-ai",   // Replit's AI package
	"google.cloud": "google-cloud", // GCP SDK
```
https://linear.app/replit/issue/DX-77/we-should-add-googlecloud-google-cloud-to-the-package-guessing